arkouda/categorical.py

```diff
@@ -137,6 +137,8 @@ def __init__(self, values, **kwargs) -> None:
         self.nlevels = self.categories.size
         self.ndim = self.codes.ndim
         self.shape = self.codes.shape
+        self.dtype = np.str_
+        self.objType = "Categorical"
```

Comment:
Let's change the attribute name to `objtype` (lowercase) to keep consistent with `pdarray`: 

https://github.com/Bears-R-Us/arkouda/blob/9b400e6560d1fccac3206f4f74f01ba0fc70758e/arkouda/pdarrayclass.py#L163

Also, I know you literally just implemented what I wrote in the issue, but I have some feedback for myself :)
* we may want to use the `str_` dtype object imported from `arkouda.dtypes` instead of `np.str_`, just to keep everthing coming from one place
* I like @pierce314159 's proposal to use `self.__class__.__name__` as the objtype instead of hard-coding the name

Reply:
Updated

Reply:
Thanks!  However, it looks like `self.objType` is still camel case (should be `self.objtype`)... or am I missing a commit?